source/GlassView.Export.Test/Convenience.cs

```diff
@@ -1,3 +1,4 @@
+using System.IO;
```

Comment:
From this, I deduce that you only removed System.IO from the global usings. It's still possible to reference it directly somewhere in your code.
Wouldn't it make sense to create a unit test, with reference checks to ensure that only in the allowed files, System.IO is present?

Reply:
Sure, we can do that anytime. I only wanted to demonstrate my take on a file system abstraction to you (Atmoos.World). Checking for that namespace and failing unit tests because of it is IMHO something we should (or shouldn't) do irrespective of Atmoos.World...

PS: Also, this is still a mini-project with two developers at most. We can IMHO also just agree to not reference System.IO and call it a day :-)
Should the project ever grow beyond just the two of us, of course we can install all sorts of fun architectural enforcement rules ;-)